src/v2/spec.yaml

```diff
@@ -671,7 +677,18 @@ components:
       properties:
         user:
           $ref: "#/components/schemas/User"
-
+    UserReactionsResponse:
+      type: object
+      required:
+        - reactions
+        - next
+      properties:
+        reactions:
+          type: array
+          items:
+            $ref: "#/components/schemas/NotificationReactions"
```

Comment:
hmm..this shouldn't be named `NotificationReactions` if it's going to be used in a non-notification API. Can we change the name to be just `Reactions`?